### PR TITLE
testonly: delete unused code

### PR DESCRIPTION
**Pull request by countvonzero**
## Motivation

TEST

## TODO

- [ ] Explain motivation or link existing issue(s)
- [ ] Test changes and document test plan
- [ ] Update documentation as needed

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

**Comment by countvonzero** [comment]
bors try

File: cmd/node/node.go [hunk 1]
```diff
@@ -296,34 +295,6 @@ func (app *App) introduction() {
 	log.Info("Welcome to Spacemesh. Spacemesh full node is starting...")
 }
 
-type clockErrorDetails struct {
```
Comment (dshulyak) [inline cmd/node/node.go:299]: hm we actually should use this error for peersync failure

Comment (dshulyak) [reply cmd/node/node.go:299]: i think Nikita used it only for ntp which was a mistake

**Comment by dshulyak** [comment]
does it break something? i would rather open an issue so that this error is used by peersync verifier. smapp expects this error to show meaningful error to the user

**Comment by countvonzero** [comment]
> does it break something? i would rather open an issue so that this error is used by peersync verifier. smapp expects this error to show meaningful error to the user

no. i was using this PR to test CI. feel free to open an issue

**Pull request closed without merging**